draft-ietf-aipref-vocab.md

```diff
@@ -46,28 +46,24 @@ informative:
 
 --- abstract
 
-This document proposes a standardized vocabulary for expressing preferences related to how content is consumed by automated processing systems.
-This vocabulary allows for the creation of structured declarations about restrictions or permissions for use of digital assets.
-The vocabulary is agnostic to the means by which it is conveyed.
-The definitions in the vocabulary enable a shared understanding between entities that express their preference and those that consume the associated digital assets.
+This document proposes a standardized vocabulary for expressing preferences related to how digital assets are used by automated processing systems. 
+This vocabulary allows for the creation of structured declarations about restrictions or permissions for use of digital assets by such systems. 
+The vocabulary is agnostic to the means by which it is conveyed. 
+The definitions in the vocabulary facilitate a shared understanding between entities that express such preferences and those that use the associated digital assets.
 
 --- middle
 
 # Introduction
 
-This document defines a common vocabulary of terms for automated systems that process digital assets.
-The primary purpose of this vocabulary is to enable machine-readable expressions of preferences about how digital assets are used by automated processing systems, such as AI training and other forms of Text and Data Mining (TDM).
+This document defines a common vocabulary of terms for automated systems that process digital assets. The primary purpose of this vocabulary is to enable machine-readable expressions of preferences about how digital assets are used by automated processing systems, in the context of training AI models and other forms of text and data mining (TDM).
 
-The elements of the vocabulary can be used to describe, in a standardized way, the types of uses that a declaring party may wish to explicitly restrict or allow.
-This ensures that preferences - either to indicate an opt-out or to explicitly admit a given usage - can be communicated, processed and stored in a consistent and interoperable manner.
+The terms defined by the vocabulary can be used to describe, in a standardized way, the types of uses that a declaring party may wish to explicitly restrict or allow. This ensures that preferences—expressed as restrictions or permissions concerning specific types of use defined by the vocabulary—can be communicated, processed, and stored in a consistent and interoperable manner.
```

Comment:
The middle clause of this second sentence seems a little rough to me.  Perhaps:

```suggestion
The terms defined by the vocabulary can be used to describe, in a standardized way, the types of uses that a declaring party may wish to explicitly restrict or allow.
Preferences are then expressed as a grant or denial of permission concerning each of the types of use defined in the vocabulary.
This ensures that preferences can be communicated, processed, and stored in a consistent and interoperable manner.
```

(I'm moving to sentence-per-line as much as possible so that these sorts of changes are easier to manage. The underlying format doesn't really care what we do, so I'm OK with anything really.)